From 80f7e1c144103d52a877f2d42db6a427867591f6 Mon Sep 17 00:00:00 2001
From: Ondrej Holy <oholy@redhat.com>
Date: Thu, 14 Jun 2018 15:22:15 +0200
Subject: [PATCH] build: Fix gschema translations
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

After meson port, translations are merged in gschema file, which doesn't
work:

$ LANG=en_US gsettings describe org.gnome.baobab.preferences excluded-uris
Una lista d'URI t'as particiones que cal excluir de l'analï¿½s.

Translations should not be merged in gschema files and also I don't see any
other gschema file with merged translations in /usr/share/glib-2.0/schemas/.
Let's install gschema file as is. After this change, translations works as
expected.
---
 data/meson.build                                           | 7 ++-----
 ....baobab.gschema.xml.in => org.gnome.baobab.gschema.xml} | 0
 po/POTFILES.in                                             | 2 +-
 3 files changed, 3 insertions(+), 6 deletions(-)
 rename data/{org.gnome.baobab.gschema.xml.in => org.gnome.baobab.gschema.xml} (100%)

diff --git a/data/meson.build b/data/meson.build
index 5b1ef00..98d1d69 100644
--- a/data/meson.build
+++ b/data/meson.build
@@ -22,11 +22,8 @@ appdata_file = i18n.merge_file(
   install_dir: join_paths(get_option('datadir'), 'metainfo'),
 )
 
-gschema_file = i18n.merge_file(
-  input: 'org.gnome.baobab.gschema.xml.in',
-  output: 'org.gnome.baobab.gschema.xml',
-  po_dir: '../po',
-  install: true,
+install_data (
+  'org.gnome.baobab.gschema.xml',
   install_dir: join_paths(get_option('datadir'), 'glib-2.0', 'schemas')
 )
 
diff --git a/data/org.gnome.baobab.gschema.xml.in b/data/org.gnome.baobab.gschema.xml
similarity index 100%
rename from data/org.gnome.baobab.gschema.xml.in
rename to data/org.gnome.baobab.gschema.xml
diff --git a/po/POTFILES.in b/po/POTFILES.in
index 0c9452a..f3d6521 100644
--- a/po/POTFILES.in
+++ b/po/POTFILES.in
@@ -2,7 +2,7 @@
 # Please keep this file sorted alphabetically.
 data/org.gnome.baobab.appdata.xml.in
 data/org.gnome.baobab.desktop.in
-data/org.gnome.baobab.gschema.xml.in
+data/org.gnome.baobab.gschema.xml
 src/baobab-application.vala
 src/baobab-cellrenderers.vala
 src/baobab-location-list.ui
-- 
2.21.0

