diff --git a/src/baobab-window.vala b/src/baobab-window.vala
index c91bf46..8dc0c94 100644
--- a/src/baobab-window.vala
+++ b/src/baobab-window.vala
@@ -311,6 +311,7 @@ namespace Baobab {
                                    "logo-icon-name", "baobab",
                                    "version", Config.VERSION,
                                    "comments", _("A graphical tool to analyze disk usage."),
+                                   "website", "https://wiki.gnome.org/action/show/Apps/DiskUsageAnalyzer",
                                    "copyright", copyright,
                                    "license-type", Gtk.License.GPL_2_0,
                                    "wrap-license", false,

